frontend/src/scenes/experiments/SecondaryMetrics.tsx

```diff
@@ -97,9 +97,9 @@ export function SecondaryMetrics({
                     <div>
                         {item.results?.[idx].result ? (
                             item.results[idx].insightType === InsightType.FUNNELS ? (
-                                <>{(item.results[idx].result * 100).toFixed(1)}%</>
+                                <>{((item.results[idx].result as number) * 100).toFixed(1)}%</>
```

Comment:
I know we might do it in some places 😱 but `as type` is generally not recommended because while it clears annoying type errors, it doesn't fix the problem which is somewhere in type definitions `result` of `item.results[idx].result` is not returning as a number

Reply:
if you look at line 98, it handles the case where it's undefined, but type parsing isn't inferring that for some reason. Do you know why?

Reply:
(this handles the undefined case, typing says its number | undefined when it should be number here)

Reply:
hmm yes that's weird 🤔 

this seems to work though, and slightly cleaner? 

<img width="729" alt="Screen Shot 2023-04-13 at 1 20 10 PM" src="https://user-images.githubusercontent.com/25164963/231836165-db989640-3813-4f0e-9015-7e032f220d37.png">